feature/onboarding/src/main/java/dev/arkbuilders/rate/feature/onboarding/pages/OnboardingScreen.kt

```diff
@@ -83,7 +84,10 @@ fun OnboardingScreen(navigator: DestinationsNavigator) {
             onClick = {
                 if (pageState.currentPage + 1 < items.size) {
                     scope.launch {
-                        pageState.animateScrollToPage(pageState.currentPage + 1)
+                        pageState.animateScrollToPage(
+                            pageState.currentPage + 1,
+                            animationSpec = tween(durationMillis = 1000),
+                        )
```

Comment:
`spring` animation spec is used by default here
I think a second is too long

Reply:
> `spring` animation spec is used by default here I think a second is too long

But I thought `1000` looked more smooth like the scrolling. Yeah, `spring` is the default, but we are using `tween` here